src/fs/fs.go

```diff
@@ -131,3 +128,60 @@ func IsPackage(buildFileNames []string, name string) bool {
 	}
 	return false
 }
+
+// Try to gracefully rename the file as the os.Rename does not work across
+// filesystems and on most Linux systems /tmp is mounted as tmpfs
+func renameFile(from, to string) (err error) {
+	err = os.Rename(from, to)
+	if err == nil {
+		return err
```

Comment:
nit: reads a bit better if we return nil here. I thought the if statement was `err != nil` like it normally would be. Returning nil would make it clear this is the happy path.

Reply:
done.